src/components/radio/docs/Radio.md

```diff
@@ -0,0 +1,96 @@
+## Radio button
+
+Seleção única entre múltiplas opções.
+
+### Uso básico
+
+Radio buttons não devem ter muitas opções. Se este for o caso, prefira o componente Select.
```

Comment:
Não seria?

Radio buttons não devem ter muitas opções. Se este for o caso, prefira o componente **Checkbox**.